package.json

```diff
@@ -34,7 +34,7 @@
     "highlight.js": "^9.14.2",
     "htmlparser2": "^3.10.1",
     "ignore": "^5.1.4",
-    "js-beautify": "~1.7.5",
+    "js-beautify": "^1.10.1-rc2",
```

Comment:
Are there breaking changes from `1.10.1-rc2` to `1.10.3`?

Reply:
upgraded to `1.10.3`.